src/hooks/query/useFetchReviewsScore.tsx

```
@@ -0,0 +1,28 @@
+		| 'OFFICE_STRETCHING'
+		| 'MINDFULNESS'
+		| 'WORKATION';
+
+	const { data, isLoading, error } = useQuery<ReviewScore, Error>({
+		queryKey: ['reviewScores', type],
+		queryFn: () => getReviewScore({ gatheringId, type }),
+		enabled: !!type,
+	});
+
+	if (error) {
+		console.log(error);
```

ESLint

[error] Unexpected console statement.

(no-console)